backend/app/main.py

```
@@ -0,0 +1,52 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.staticfiles import StaticFiles
```

Ruff (0.8.2)

`fastapi.staticfiles.StaticFiles` imported but unused

Remove unused import: `fastapi.staticfiles.StaticFiles`

(F401)